hibernate-reactive-core/src/test/java/org/hibernate/reactive/CompletionStagesTest.java

```diff
@@ -0,0 +1,177 @@
+/* Hibernate, Relational Persistence for Idiomatic Java
+ *
+ * SPDX-License-Identifier: LGPL-2.1-or-later
+ * Copyright: Red Hat Inc. and Hibernate Authors
+ */
+package org.hibernate.reactive;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.IntStream;
+
+import org.hibernate.reactive.util.impl.CompletionStages;
+
+import org.junit.Test;
+
+import org.assertj.core.api.Assertions;
+
+public class CompletionStagesTest {
+
+	static int loopCount = 0;
+
+	@Test
+	public void testTotalWithIntegers() {
+		// total(int start, int end, Function<Integer,CompletionStage<Integer>> consumer)
+		int startInt = 0;
+		int endInt = 11;
+
+		CompletionStages.total(
```

Comment:
I think it would be more readable and compact if you put a static import of `CompletionStages`

Reply:
thx for the feedback

Reply:
Done

Reply:
done